bots/validatespeakerscornerissue.py

```diff
@@ -0,0 +1,118 @@
+import os
+from datetime import date, timedelta
+from github import Github
+import arxiv
+
+VSF_BOT_TOKEN = os.getenv("VSF_BOT_TOKEN")
+ISSUENUMBER = os.getenv("ISSUENUMBER")
+
+g = Github(VSF_BOT_TOKEN)
+repo = g.get_repo("virtualscienceforum/virtualscienceforum")
+issue = repo.get_issue(number=ISSUENUMBER)
+
+def CheckDate(dateStr, timeStr):
+    scheduledDate = date.fromisoformat(dateStr)
+    if( (scheduledDate - date.today()).days < 14 ):
```

Comment:
I suggest to use `datetime` objects, not dates.  Also this line is nicer done like this (not converting to integers):

```suggestion
    if scheduled - datetime.now() < timedelta(days=14):
```

Reply:
Thanks for catching this! I had imported timedelta and had meant to do this.